src/app-nav-bar/user-menu/user-menu.js

```diff
@@ -35,7 +35,7 @@ export default function UserMenu(props: AppNavBarPropsT) {
       triggerType={TRIGGER_TYPE.click}
     >
       <Button overrides={{BaseButton: {component: StyledUserMenuButton}}}>
-        <Avatar name={username} src={userImgUrl} size={'32px'} />
+        <Avatar name={username || ''} src={userImgUrl} size={'32px'} />
```

Comment:
in the case where the username is `undefined`, do we want to have an empty string as the prop value? or should we remove the prop entirely?

Reply:
`name` is a required prop in `Avatar` (could be replaced with custom `initials`) and it doesn't provide a default value, therefore, errors out.